test/e2e/v3/generate_test.go

```diff
@@ -398,3 +398,101 @@ Count int `+"`"+`json:"count,omitempty"`+"`"+`
 #          create: true`, "#")).To(Succeed())
 
 }
+
+// GenerateV4 implements a go/v4(-alpha) plugin project defined by a TestContext.
+func GenerateV4(kbc *utils.TestContext, crdAndWebhookVersion string) {
+	var err error
+
+	By("initializing a project")
+	err = kbc.Init(
+		"--plugins", "go/v4-alpha",
```

Comment:
The v4 tests myd be in test/e2e/v4 instead 
Could you please ensure that we have a new dir with them?